GoogleTestAdapter/EmptyExe/EmptyExe.c

```diff
@@ -0,0 +1,2 @@
+// Compile with "cl /Zl" to have a "valid" but almost empty executable file (no code, no imports, data segment with only zeros). 
```

Comment:
It's crazy that this is possible at all :-)

Reply:
Yes.  My first version was in assembly, to only have symbols without anything else, but I then thought it would "simpler" in C.  In the C version, the compiler generates a ".data" section (of one byte) which is not in the ASM version, but this is not a problem in our case.